ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -67,14 +69,38 @@ export class TableOptions extends PureComponent<Props, {}> {
     return tableOptionsDifferent
   }
 
+  public moveField(dragIndex, hoverIndex) {
```

Comment:
if moveField is being defined here and passed down to children components, couldn't it be private?